server/pulp/server/db/querysets.py

```diff
@@ -116,6 +119,27 @@ def get_or_404(self, **kwargs):
             raise pulp_exceptions.MissingResource(**kwargs)
 
 
+class WorkerQuerySet(CriteriaQuerySet):
+    """
+    Custom queryset for workers
+    """
+
+    def get_workers_online(self):
+        """
+        Returns a queryset that contains only worker documents that have been updated in the last
```

Comment:
The first line of a multiline docstring should be a single line. Please shorten this to 1 line, or add a shorter 1 line above with a whitespace between. See the PEP here:  https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings